local-cli/generator/templates/_buckconfig

```diff
@@ -0,0 +1,9 @@
+
+[android]
+  target = Google Inc.:Google APIs:23
+
+[maven_repositories]
+  central = https://repo1.maven.org/maven2
+
+[alias]
+  android-app = //android/app:app
```

Comment:
You can do `buck install -r //android/app:app`, but also `buck install -r android/app` so not sure the alias `android-app` adds much value.

https://buckbuild.com/concept/build_target.html